common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -142,7 +142,7 @@ private boolean handleUrl(final WebView view, final String url) {
             if (isPkeyAuthUrl(formattedURL)) {
                 Logger.info(TAG, "WebView detected request for pkeyauth challenge.");
                 final PKeyAuthChallengeFactory factory = new PKeyAuthChallengeFactory();
-                final PKeyAuthChallenge pKeyAuthChallenge = factory.getPKeyAuthChallenge(url);
+                final PKeyAuthChallenge pKeyAuthChallenge = factory.getPKeyAuthChallengeFromWebViewRedirect(url);
```

Comment:
Hmm. this method should be covered by the new test class `PKeyAuthChallengeFactoryTest`.

@paulkagiri do you have any idea what might have happened?